backend/utils/file_cleanup.py

```
@@ -0,0 +1,416 @@
+"""
+Conservative file cleanup utilities for StudyMate v2.
+Focuses on safety and configurable cleanup with multiple safeguards.
+"""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)